learning_cp/eternity2/features2.jl

```diff
@@ -0,0 +1,77 @@
+struct EternityFeaturization <: SeaPearl.AbstractFeaturization end
+"""
+node features:
+1 => node is id
+2 => node is horizontal edge
+3 => node is vertical edge
+4 => vertical index
+5 => horizontal index
+6 => domain length
+7 => # constraints linked
+8 => assigned value
+9 => node is constraint
+10 => node value
+11 => node degree
+
+global features:
+None
+"""
+
+function SeaPearl.featurize(sr::SeaPearl.DefaultStateRepresentation{EternityFeaturization,TS}) where TS
+    n = eternity2_generator.n
+    m = eternity2_generator.m
+    k = eternity2_generator.k
+    g = sr.cplayergraph
+    features = zeros(Float32, 11, nv(g))
+    for i in 1:nv(g)
+        cp_vertex = SeaPearl.cpVertexFromIndex(g, i)
+        if isa(cp_vertex, SeaPearl.VariableVertex)
+            id = cp_vertex.variable.id
+            #println(id)
+            prefix, suffix = split(id,'_')
+            if prefix=="id"
+                features[1,i] = 1
+            elseif prefix=="srch"
+                features[2,i] = 1 
+            elseif prefix=="srcv"
+                features[3,i] = 1 
+            else
+                println("WTF ??")
```

Comment:
ಠ_ಠ

Reply:
This whole file was created for testing purpose only and shouldn't be merged to master